src/FusionAuthClient.ts

```diff
@@ -2678,10 +2690,14 @@ export class FusionAuthClient {
   private start(): IRESTClient {
     let client = this.clientBuilder.build(this.host).withAuthorization(this.apiKey);
 
-    if (this.tenantId !== null && typeof(this.tenantId) !== 'undefined') {
+    if (this.tenantId != null) {
```

Comment:
This is the more succinct way to check `null` or `undefined`and, to my knowledge, the only time you should not use `===`. Correct me if I'm wrong.

Reply:
Seems good to me.

> When comparing null and undefined, they are only equal to themselves and each other:
> ```
> null == null
> // true
> undefined == undefined
> // true
> null == undefined
> // true
> ```
> If you try to compare null to any other value, it will return false.